actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserProfileReadActor.java

```diff
@@ -298,6 +324,79 @@ private Response getUserProfileData(Request actorMessage) {
     return response;
   }
 
+  /**
+   * fetch declared info from user_declaration table
+   *
+   * @param userId
+   * @return
+   */
+  private List<Map<String, Object>> fetchUserDeclarations(String userId) {
+    Map<String, Object> propertyMap = new HashMap<>();
+    propertyMap.put(JsonKey.USER_ID, userId);
+    Response response =
+        cassandraOperation.getRecordsByProperties(
+            JsonKey.SUNBIRD, JsonKey.USR_DECLARATION_TABLE, propertyMap);
+    List<Map<String, Object>> resExternalIds;
+    List<Map<String, Object>> finalRes = new ArrayList<>();
+    if (null != response && null != response.getResult()) {
+      resExternalIds = (List<Map<String, Object>>) response.getResult().get(JsonKey.RESPONSE);
+      if (CollectionUtils.isNotEmpty(resExternalIds)) {
+        resExternalIds.forEach(
+            item -> {
+              Map<String, String> declaredFields =
+                  (Map<String, String>) item.get(JsonKey.USER_INFO);
+              if (MapUtils.isNotEmpty(declaredFields)) {
+                decryptDeclarationFields(declaredFields);
+              }
+              declaredFields.put(JsonKey.STATUS, (String) item.get(JsonKey.STATUS));
+              declaredFields.put(JsonKey.ERROR_TYPE, (String) item.get(JsonKey.ERROR_TYPE));
+              declaredFields.put(JsonKey.ORG_ID, (String) item.get(JsonKey.ORG_ID));
+              finalRes.add(
+                  new HashMap<String, Object>() {
+                    {
+                      put((String) item.get(JsonKey.ROLE), declaredFields);
+                    }
+                  });
+            });
+      }
+    }
+    return finalRes;
+  }
+
+  private Map<String, String> decryptDeclarationFields(Map<String, String> declaredFields) {
+    if (declaredFields.containsKey(JsonKey.DECLARED_EMAIL)) {
+      declaredFields.put(
+          JsonKey.DECLARED_EMAIL,
+          UserUtil.getDecryptedData(declaredFields.get(JsonKey.DECLARED_EMAIL)));
+    }
+    if (declaredFields.containsKey(JsonKey.DECLARED_PHONE)) {
+      declaredFields.put(
+          JsonKey.DECLARED_PHONE,
+          UserUtil.getDecryptedData(declaredFields.get(JsonKey.DECLARED_PHONE)));
+    }
```

Comment:
This happened just yesterday - We are not going to have district, state as part of declaration. 
Also, transforming uuid to code or viceversa is not a fitting action to be performed within a decrypt() function. Pls remove the state/district.

Reply:
removed the state/district.